test/artefacts/marriage-abroad/mozambique/ceremony_country/partner_british/opposite_sex.txt

```diff
@@ -0,0 +1,13 @@
+Marriage in Mozambique
```

Comment:
It is a bit weird to me to start the commit notes with `This commit adds`. What about removing `This commit`. We already know we are in a commit.